src/client/coll/mod.rs

```diff
@@ -38,7 +38,7 @@ impl<'a> Collection<'a> {
     /// Creates a collection representation with optional read and write controls.
     ///
     /// If `create` is specified, the collection will be explicitly created in the database.
-    pub fn new(db: &'a Database<'a>, name: &str, create: bool,
+    pub fn new(db: &'a Database<'a>, name: &str, _create: bool,
```

Comment:
Pymongo provides the same option; we just haven't implemented `create_collection` yet in db.